students/ryba_paula/lesson_14_decorators/access_required.py

```diff
@@ -0,0 +1,24 @@
+from authorization import has_access
+
+
+def access_required(func):
+    def func_wrapper(*args, **kwargs):
+        if has_access():
+            return func(*args, **kwargs)
```

Comment:
jeżeli definiujesz wyjścia z funkcji (`return`) to zdefiniuj wszystkie możliwe, nawet jeśli zwracają `None`